site/en/docs/devtools/javascript/breakpoints/index.md

```diff
@@ -56,7 +56,7 @@ console.log('c');
 ### Conditional line-of-code breakpoints {: #conditional-loc }
 
 Use a conditional line-of-code breakpoint when you know the exact region of code that you need to
-investigate, but you want to pause only when some other condition is true.
+investigate, but you want to pause only when some other condition is true. This can help you catch edge cases with variables.
```

Comment:
Conditional breakpoint is useful to not annoy yourself with irrelevant break during debugging. It helps developers focus on just stopping at the condition relevant to the current debugging, especially in a loop. 

Tho it's helpful for catching edge cases but not so much on that.